packages/alert/src/lib/alert.tsx

```diff
@@ -0,0 +1,125 @@
+import MUIAlert, {
+  AlertProps as MUIAlertProps,
+  AlertColor,
+} from '@mui/material/Alert';
+import { styled } from '@mui/material/styles';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import CheckCircleIcon from '@mui/icons-material/CheckCircleOutline';
+import Box from '@mui/material/Box';
+import { Typography } from '@mui/material';
+
+/* eslint-disable-next-line */
+export interface AlertProps extends MUIAlertProps {
+  severity?: AlertColor;
+  onClose?: () => void;
+  title?: string | undefined;
+  action?: React.ReactNode;
+}
+
+const StyledAlert = styled(MUIAlert)(({ theme }) => ({
+  borderRadius: '2px',
+  marginTop: '36px',
+  marginBottom: '12px',
+  paddingTop: '6px',
+  paddingBottom: '8px',
+  display: 'flex',
+  color: theme.palette.text.primary,
+  fontSize: 'small',
+  backgroundColor: theme.palette.background.default,
+
+  '& .MuiAlert-icon': {
+    marginRight: '12px',
+    fontSize: '16px',
+  },
+
+  '& .MuiAlert-message': {
+    padding: '6px 0',
+  },
+
+  '&.MuiAlert-standardSuccess': {
+    border: `1px solid ${theme.palette.success.main}`,
+    '& .MuiAlert-icon': {
+      color: theme.palette.success.main,
+    },
+  },
+
+  '&.MuiAlert-standardWarning': {
+    border: `1px solid ${theme.palette.warning.main}`,
+    '& .MuiAlert-icon': {
+      color: theme.palette.warning.main,
+    },
+  },
+
+  '&.MuiAlert-standardError': {
+    border: `1px solid ${theme.palette.error.main}`,
+    '& .MuiAlert-icon': {
+      color: theme.palette.error.main,
+    },
+  },
+
+  '&.MuiAlert-standardInfo': {
+    border: `1px solid ${theme.palette.info.main}`,
+    '& .MuiAlert-icon': {
+      color: theme.palette.info.main,
+    },
+  },
+}));
```

Comment:
since we don't have any logic with custom props, lets move this changes to styles sheet at @novatics-ui/styles package 

https://github.com/Novatics/novatics-ui/tree/master/packages/styles/src/lib/Theme/components

Reply:
Done